doc/getting-started/Quick-setup.md

```diff
@@ -76,76 +76,157 @@ The default XMPP domain served by MongooseIM right after installation is `localh
 
 You can register (create) users with the `mongooseimctl` utility.
 
-This command registers the user `user@domain` using password `password`.
+This command registers the user `user@localhost` using password `secret`.
 ```bash
-mongooseimctl register_identified user domain password
+mongooseimctl account registerUser --username user --domain localhost --password secret
 ```
 Examples:
 ```bash
-mongooseimctl register_identified alice localhost qwerty
-mongooseimctl register_identified bob localhost 12345678
-mongooseimctl register_identified carol localhost abc123
-mongooseimctl register_identified dan localhost dan
+mongooseimctl account registerUser --username alice --domain localhost --password qwerty
+mongooseimctl account registerUser --username bob --domain localhost --password 12345678
+mongooseimctl account registerUser --username carol --domain localhost --password abc123
+mongooseimctl account registerUser --username dan --domain localhost --password dan
 ```
 !!! Warning
     The password is entered manually in the command line and history is accessible to the command line users.
     This method is not recommended for production use, you may prefer for example [LDAP](../authentication-methods/ldap.md).
 
-You can check that it has correctly been created:
+You can check that the user account has been created:
 ```bash
-mongooseimctl check_account user host
-```
-Example:
-```bash
-mongooseimctl check_account alice localhost
-mongooseimctl check_account bob localhost
-mongooseimctl check_account carol localhost
-mongooseimctl check_account dan localhost
+mongooseimctl account checkUser --user alice@localhost
+{
+  "data" : {
+    "account" : {
+      "checkUser" : {
+        "message" : "User alice@localhost exists",
+        "exist" : true
+      }
+    }
+  }
+}
 ```
 
 Now you can list all registered users in your host:
 ```bash
-mongooseimctl registered_users host
-```
-Example:
+mongooseimctl account listUsers --domain localhost
+{
+  "data" : {
+    "account" : {
+      "listUsers" : [
+        "alice@localhost",
+        "bob@localhost",
+        "carol@localhost",
+        "dan@localhost"
+      ]
+    }
+  }
+}
+```
+
+If you want to delete users in your host:
```

Comment:
```suggestion
If you want to delete user in your host:
```

Reply:
In my opinion using "users" suggest possibility of deleting multiple users using that query.